### ISSUE TITLE
Article Style Guide is still using the Mediawiki Guidelines we have

### ISSUE THREAD

**Issue by sr229** [ppy/osu-wiki#104]
We must rewrite completely the Article Style Guides in accordance with #6 since we are no longer using MW.

**Comment by iamanai** [comment]
Since all wiki editing is done on GitHub now, I feel that a dedicated "style guide" wiki page is no longer necessary. I think #6 itself could be our style guide going forward.

**Comment by MegaApplePi** [comment]
#6 is technically the draft of the Article Style Guide. I would prefer to have a dedicated manual of style page just to have a functioning TOC.
RaikireHiuduo is practically the one holding the manual of style together with string and duct tape. This means that no one but him can edit it.

**Comment by iamanai** [comment]
Oh, I didn't mean keep the issue itself as the guide. I meant use what he made in the issue as the guide. To me, it would make sense to make it a file in the master directory (alongside readme) so it's not inside the wiki itself. We will never be reading it from the wiki itself so there's no reason to have it there.

**Comment by RaikireHiuduo** [comment]
Just a quick note that if the Article Styling Guide (or #102 ) was done _outside of the wiki folder_, you will need to manually add a TOC and read from GitHub itself _only_ (which is not as bad as it sounds like).

The new site only reads from the `wiki` folder, and searches for `en.md` (assumed no locales) inside the destination folder (like, say Game_Modes: https://new.ppy.sh/wiki/Game_Modes/).

Alternative route is to add a tab called "Wiki" on the osu-wiki repository (which is not a really good suggestion since it force the content in the Wiki tab to EN-only as far as I know).

I agree with @iamanai suggestion to put it outside the `wiki` folder.

**Comment by iamanai** [comment]
I'll give it a go and submit a PR, unless there are any objections.

**Comment by RaikireHiuduo** [comment]
...Or a third solution (which I am preferring right now). Still keep the `Article_Styling_Guide` folder and `Getting_Started_Contribute` folder into the `wiki` folder, but on the root folder, modify the `README.md` with some explanation and link to the two instead. This will remove the TOC necessity and nicer CSS.

Something like this added to the `README.md` (which is incomplete; write a better one-liner or some lines, please):-

```
## Contribution guide
If you are new to GitHub, visit [Getting Started Contribute](/wiki/Getting_Started_Contribute "Getting Started Contribute").

Please refer to [Article Styling Guide](/wiki/Article_Styling_Guide "Article Styling Guide") for quick help in writing.
```

**Comment by sr229** [comment]
Issue Closed by #129
